File: public/usage-examples/generative_AI/generate_text-1-example.py
from splashkit import *

# Prompt sent to the AI model
prompt = "Give me a fun fact about space."

# Let the user know the AI response is being generated
write_line("Generating AI response...")
write_line("")

# Generate a text response from the AI
response = generate_text(prompt)

# Display the prompt and AI response
write_line("Prompt: " + prompt)
write_line("")
write_line("AI Response:")
write_line(response)

File: public/usage-examples/generative_ai/generate_text-1-example.cpp
#include "splashkit.h"

int main()
{
    // Define a prompt to send to the AI
    // Note: Requires a local AI model to be set up via SplashKit
    string prompt = "What is the capital of France?";
    write_line("Prompt: " + prompt);

    // Generate a text response from the AI
    string response = generate_text(prompt);
    write_line("Response: " + response);

    return 0;
}